Makefile.toml

```diff
@@ -79,6 +80,14 @@ args = [
     "-D", "warnings"
 ]
 
+# Overridden because: workspace = false and custom toolchain
+[tasks.format]
+workspace = false
+clear = true
+script = [
+  "cargo +nightly-2019-04-30 fmt"
```

Comment:
🔧 same than above. maybe have a file called `rust-toolchain-for-fmt` containing `nightly-2019-04-30` (with no end of line :) )

Reply:
If the toolchain where to change often, yeah maybe. But I don't see a reason why it would change ever (apart from moving to stable, which would make the file obsolete or new formatting rules/changes that are only present in a newer nightly version). The only reason we have to pin it is because not every nightly has `rustfmt`, otherwise we could just go for the latest nightly.

Given that, I actually like having one file _less_ in the repository and some minor duplication within several configuration files. If any of that is out of sync, you immediately notice it anyway because the build on CI fails.